apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-tabs.tsx

```diff
@@ -51,16 +52,21 @@ export const TestWorkflowTabs = ({ testData }: { testData: WorkflowTestDataRespo
       showToast({
         variant: 'lg',
         children: ({ close }) => (
-          <SuccessButtonToast
-            title="Test workflow triggered successfully"
-            description={`Test workflow ${workflowSlug} was triggered successfully`}
-            actionLabel="View activity feed"
-            onAction={() => {
-              close();
-              navigate(buildRoute(LEGACY_ROUTES.ACTIVITY_FEED, { transactionId }));
```

Comment:
The navigate here works like an internal routing, there is no option to do a full page reload on navigate.